tracer/test/snapshots/Security.AspNetMvc5.Integrated.enableSecurity=True.__test=server.request.query_url=_Health_-arg=[$slice]_body=null.verified.txt

```diff
@@ -18,7 +18,8 @@
       http.url: http://localhost/Health/?arg=[$slice],
       http.useragent: Mistake Not...,
       language: dotnet,
-      span.kind: server
+      span.kind: server,
+      _dd.origin: appsec
```

Comment:
I don't understand why the Modified snapshots now have a `_dd.origin: appsec`, do you know what change caused this?

Reply:
Yes, there was quite a long internal discussion about this. This is a child span. The WAF is called earlier, before this child span is created. When the child span is created, the `_dd.origin` tag is copied from the parent to the child. It's not copied if the span is tagged after the child created, thus the difference.

We could write some code to stop the copying in these circumstances, but since we need to tag the root spans and it doesn't really matter if other spans are tagged or not, we decided to accept the change.